src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandler.java

```diff
@@ -298,6 +298,8 @@ private PreSubmitCallbackResponse<SscsCaseData> buildResponse(Callback<SscsCaseD
                                                                   DocumentLink url,
                                                                   SscsDocumentTranslationStatus documentTranslationStatus) {
 
+        clearInterlocReferralReason(caseData);
```

Comment:
I would put a flag around this and check what has been selected from the drop down, from this ticket it seems like we should only clear it if one of the post hearing ones has been selected

Reply:
PH A flag added.  Makeda has confirmed that any pre-existing reason should be cleared, not just PH ones.